ecrypto/ecrypto.go

```diff
@@ -78,6 +78,9 @@ func Decrypt(ciphertext []byte, key []byte) ([]byte, error) {
 }
 
 // SealWithUniqueKey encrypts a given plaintext with a key derived from a measurement of the enclave.
+//
+// Ciphertexts can't be decrypted if the UniqueID of the enclave changes. If you want
+// to be able to decrypt ciphertext across enclave versions, use SealWithProductKey.
 func SealWithUniqueKey(plaintext []byte) ([]byte, error) {
```

Comment:
Maybe you could add a short paragraph to the ecrypto.doc file describing in which scenarios/ for what tasks those functions are used.